src/components/QuranReader/TranslationView/hooks/useGetVersesCount.ts

```diff
@@ -0,0 +1,48 @@
+import { useContext, useMemo } from 'react';
+
+import { useSelector } from 'react-redux';
+
+import useFetchPagesLookup from '../../hooks/useFetchPagesLookup';
```

Comment:
nit-picking: If we have 2 levels or more we should use `@/components/QuranReader/hooks/useFetchPagesLookup.ts` import syntax instead of `../../`